Chat: Add Google Dialogflow Integration topic (#8231)
DevExpress/devextreme-documentation: concepts/05 UI Components/Chat/15 Integrate with AI Service/05 OpenAI/20 Additional Configuration.md

> You can implement additional UI capabilities to further improve user experience:

Reviewer: ```suggestion
You can add UI features to improve user experience:
```

> You can also implement additional UI capabilities to further improve user experience:

Reviewer 2: Both "also" and "additional" seem redundant: 

```suggestion
You can implement additional UI capabilities to further improve user experience:
```

> - Add a Markdown converter for assistant outputs. For more information, refer to the [Markdown Support](/concepts/05%20UI%20Components/Chat/13%20Markdown%20Support.md '/Documentation/Guide/UI_Components/Chat/Markdown_Support/') help topic.

Reviewer 2: ```suggestion
- Add a Markdown converter for assistant output. For more information, refer to the [Markdown Support](/concepts/05%20UI%20Components/Chat/13%20Markdown%20Support.md '/Documentation/Guide/UI_Components/Chat/Markdown_Support/') help topic.
```

> - Define a [messageTemplate](/api-reference/10%20UI%20Components/dxChat/1%20Configuration/messageTemplate.md '/Documentation/ApiReference/UI_Components/dxChat/Configuration/#messageTemplate') for assistant responses and add two buttons: **Copy** and **Regenerate Response**. See the example code in the GitHub repository:

Reviewer: ```suggestion
- Define a [messageTemplate](/api-reference/10%20UI%20Components/dxChat/1%20Configuration/messageTemplate.md '/Documentation/ApiReference/UI_Components/dxChat/Configuration/#messageTemplate') for assistant responses and add the **Copy** and **Regenerate Response** buttons. See the example code in the GitHub repository:
```

> - Define a [messageTemplate](/api-reference/10%20UI%20Components/dxChat/1%20Configuration/messageTemplate.md '/Documentation/ApiReference/UI_Components/dxChat/Configuration/#messageTemplate') for the assistant’s responses and add two buttons: copy and regenerate response. See the example code in the GitHub repository:

Reviewer 2: ```suggestion
- Define a [messageTemplate](/api-reference/10%20UI%20Components/dxChat/1%20Configuration/messageTemplate.md '/Documentation/ApiReference/UI_Components/dxChat/Configuration/#messageTemplate') for assistant responses and add two buttons: **Copy** and **Regenerate Response**. See the example code in the GitHub repository:
```
